cmd/neofs-cli/modules/control/shards_set_mode.go

```diff
@@ -36,12 +83,15 @@ func initControlSetShardModeCmd() {
 	flags := setShardModeCmd.Flags()
 	flags.StringSlice(shardIDFlag, nil, "List of shard IDs in base58 encoding")
 	flags.Bool(shardAllFlag, false, "Process all shards")
+
+	modes := make([]string, 0)
+
+	iterateSafeShardModes(func(strMode string) {
+		modes = append(modes, "'"+strMode+"'")
```

Comment:
use `fmt` pkg here?

Reply:
It is shorter like this

Reply:
yes but looks not that obvious (more "nonhuman" symbols like `"`, `+` IMO)

Reply:
Here we just wanna wrap string into apostrophies, I don't think formatting is worth it.